src/index.js

```diff
@@ -0,0 +1,64 @@
+// Магазин
+
+import { Client } from './Client.js';
+import { Product } from './Product.js';
+import { Shop } from './Shop.js';
+
+let danylo = new Client(1, 'Danylo', 'Varlyhin');
+let yaroslav = new Client(2, 'Yaroslav', 'Isaenkov');
+
+let potato1 = new Product(1, 'Potato', 'Vegetables', 7);
+let potato2 = new Product(2, 'Potato', 'Vegetables', 7);
+let potato3 = new Product(3, 'Potato', 'Vegetables', 7);
+
+let tomato1 = new Product(4, 'Tomato', 'Vegetables', 19);
+let tomato2 = new Product(5, 'Tomato', 'Vegetables', 19);
+let tomato3 = new Product(6, 'Tomato', 'Vegetables', 19);
+let tomato4 = new Product(7, 'Tomato', 'Vegetables', 19);
+let tomato5 = new Product(8, 'Tomato', 'Vegetables', 19);
+
+let apple1 = new Product(9, 'Apple', 'Fruits', 20);
+let apple2 = new Product(10, 'Apple', 'Fruits', 20);
+
+let banana1 = new Product(11, 'Banana', 'Fruits', 25);
+let banana2 = new Product(12, 'Banana', 'Fruits', 25);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Varlyle/my-first-project/pullRequest?prid=7306919)
```suggestion
let banana2 = new Product(12, "Banana", 'Fruits', 25);
```